src/components/app/MenuButtons/index.js

```diff
@@ -176,22 +176,19 @@ class MenuButtonsImpl extends React.PureComponent<Props> {
   render() {
     return (
       <>
-        {/* Place the info button outside of the menu buttons to allow it to shrink. */}
```

Comment:
This is not true anymore, because the string is now fixed (or fairly fixed). So I removed this container `menuButtons`.